server/src/lib/routes/verify/Get.ts

```diff
@@ -24,42 +25,43 @@ async function verifyWithSelectedMethod(req: Express.Request, res: Express.Respo
   }
 }
 
-/**
- * The Redirect header is used to set the target URL in the login portal.
- *
- * @param req The request to extract X-Original-Url from.
- * @param res The response to write Redirect header to.
- */
-function setRedirectHeader(req: Express.Request, res: Express.Response) {
```

Comment:
This is breaking, please remove the use of **upstream_http_redirect** in nginx configuration and replace with **$scheme://$http_host$request_uri;**